.dockerignore

```diff
@@ -1,6 +1,16 @@
-node_modules
-npm-debug.log
-Dockerfile
+node_modules/
+test-output/
 .dockerignore
 auth
 test-output
+
+helm
+scripts
+test-output
+docker-compose*
+Jenkinsfile
+License
+README.md
+app/static/stylesheets/*.css*
```

Comment:
Not copying the CSS files to the image means the application running in the container doesn't have any CSS files.  I'm not saying that copying them over is the right approach because I don't think it is as it's dependent on the developer building them first.  Instead, I think the image should build the CSS files during the docker build.

![image](https://user-images.githubusercontent.com/42574977/77155923-38df0500-6a96-11ea-91c7-563cd6649004.png)

Reply:
Is that a local build issue? The files are present on the deployed application at https://ffc-demo-pr108.ffc-dev.aws-int.defra.cloud/

The Dockerfile does compile the css as part of the build process as seen here 
https://github.com/DEFRA/ffc-demo-web/blob/4adf61ce4bde7d64141fefb64fd7e18224b7a706/Dockerfile#L18 

I think maybe this problem is seen when you develop locally as you are mounting the local app folder which doesn't contain the css if you haven't run the `npm run build`.

Reply:
I see.  In that case we should make the readme state that to run the application using Docker locally it's important to have run the build first.

Reply:
Thanks, I've updated the README